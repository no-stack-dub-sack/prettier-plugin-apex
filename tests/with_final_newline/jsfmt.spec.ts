runSpec(__dirname, ["apex"], { apexInsertFinalNewline: true });
