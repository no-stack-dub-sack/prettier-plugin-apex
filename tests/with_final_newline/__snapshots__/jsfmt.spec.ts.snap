// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Format apex: WithFinalNewLine.cls: WithFinalNewLine.cls 1`] = `
class WithFinalNewLine {}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
class WithFinalNewLine {
}

`;
